plugins/pulp_docker/plugins/registry.py

```diff
@@ -317,7 +319,19 @@ def get_manifest(self, reference):
         """
         path = self.MANIFEST_PATH.format(name=self.name, reference=reference)
         headers, manifest = self._get_path(path)
-        return headers['docker-content-digest'], manifest
+
+        digest_header = 'docker-content-digest'
+        if digest_header in headers:
+            digest = headers[digest_header]
+        else:
+            decoded_manifest = json.loads(manifest, object_pairs_hook=collections.OrderedDict,
```

Comment:
I can't believe I didn't think to check for this myself, but @jluza pointed out to me that Python 2.6 does not support this object_pairs_hook argument. I will have to come up with another option.